bot_globals.py

```diff
@@ -36,6 +36,26 @@
 
 RATINGS = {}
 
+MILESTONE_ROLES = {
+    1: ("Initiate (1 Point)", discord.Color.blue()),          # 1 Problem
+    10: ("Problem Solver (10 Points)", discord.Color.green()),  # 10 Problems
+    25: ("Algorithm Apprentice (25 Points)", discord.Color.gold()),  # 25 Problems
+    50: ("Logic Guru (50 Points)", discord.Color.purple()),     # 50 Problems
+    75: ("Code Crusader (75 Points)", discord.Color.orange()),  # 75 Problems
+    100: ("LeetCode Legend (100 Points)", discord.Color.red()),  # 100 Problems
+    150: ("Problem Connoisseur (150 Points)", discord.Color.teal()),  # 150 Problems
+    200: ("Mastermind (200 Points)", discord.Color.dark_blue())  # 200 Problems
```

Comment:
Suggested numbers (names and colours not yet decided/agreed-on):
1:
50: 
100:
200: 
500: 
1000: 
2000: 
4000: 
8000:

Reply:
Feel free to go wild here, not too concerned on what roles / colors end up being implemented for both, happy for you and nano to agree upon this and update it accordingly. I'm +10 GMT, so it'll probably be very difficult to refine this with me involved.